frontend/config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     // en is copied from en-US at build time, so we have to add it special
     'en,' + fs.readdirSync('./locales').join(',')  :
     // Enabled locales for stage & production - update as they reach 100%
-    'en-US,en,cs,de,dsb,es-ES,es-MX,fr,fy-NL,hsb,hu,it,ja,kab,nl,pt-BR,ru,sq,sv-SE,uk,zh-CN,zh-TW',
+    'en-US,en,cs,de,de-DE,dsb,es-ES,es-MX,fr,fy-NL,hsb,hu,it,ja,kab,nl,pt-BR,ru,sq,sv-SE,uk,zh-CN,zh-TW',
```

Comment:
Do we also need to add it to line 12 above? (although "meh" since I guess it's really only for dev server development)
```js
// en is copied from en-US at build time, so we have to add it special
```